docs/plugin_loading.md

```diff
@@ -0,0 +1,33 @@
+# Plugin Loading
+
+To load in AMP plugins, select the `Add Plugins...` option from the `Plugins` section of the upper
+taskbar.
+
+![](./pngs/add_plugins.png)
+
+Navigate to the `plugins` folder that was extracted from `plugins.zip`.  Select any of the `.plg`
+folders and hit the `Open` button to load the plugin.
+
+![](./pngs/plugins_file_explorer.png)
+
+To open and use the plugins, open the `Plugins` section of the upper taskbar and click on the
+desired plugin name.  This will open the plugin so that it can be used.
+
+![](./pngs/open_plugin.png)
+
+<div 
+    style="
+        border: 0px solid #35f;
+        border-left-width: 6px;
+        padding: 8px;
+        margin-bottom: 8px;
+        background-color: #acf;
+        color: #000"
+> 
+    <b><i>Note</i></b>:  Standard plugins are under active development.  Documentation covering
+    their usage will become available as they are completed.
+</div>
```

Comment:
A bit more whitespace between the sections so it's clear which image the text refers to